### ISSUE TITLE
Stack level too deep error

### ISSUE THREAD

**Issue by zoras** [scoutapp/scout_apm_ruby#188]
`scout_apm_ruby >= 2.4.10` throws *stack level too deep* error for all models

```ruby
$ bin/rails c
2.5.1 (main):0 > RUBY_VERSION
=> "2.5.1"
2.5.1 (main):0 > Rails.version
=> "5.2.0"
2.5.1 (main):0 > User.last
SystemStackError: stack level too deep
from /usr/local/Cellar/asdf/0.4.3/installs/ruby/2.5.1/lib/ruby/gems/2.5.0/bundler/gems/scout_apm_ruby-c06058886962/lib/scout_apm/tracked_request.rb:355:in `recorded?'
2.5.1 (main):0 >
```

**Comment by alindeman** [comment]
Hi @zoras, this is likely caused by another gem interacting with the same code path that scout_apm_ruby uses to add instrumentation. @cschneid has more details and possible workarounds over in this issue: https://github.com/scoutapp/scout_apm_ruby/issues/186#issuecomment-380837334

We're working on a fix in #187, so consider watching that pull request too.

**Comment by cschneid** [comment]
@zoras I just released version 2.4.11, can you give that a try, and if it still exhibits this error, share the whole backtrace (from `$!`)

**Comment by zoras** [comment]
@cschneid It works now! 👍
